awx_collection/README.md

```diff
@@ -32,7 +32,7 @@ Installing the `tar.gz` involves no special instructions.
 ## Running
 
 Non-deprecated modules in this collection have no Python requirements, but
-may require the official [AWX CLI](https://docs.ansible.com/ansible-tower/latest/html/towercli/index.html)
+may require the official [AWX CLI](https://docs.ansible.com/automation-controller/latest/html/controllercli/index.html)
```

Comment:
@tvo318 Hey, about the AWX CLI docs. I know there's currently no equivalent of the guide in the AWX docsite yet and we still need to migrate the content to `docs.redhat`. @samccann renewed this topic and the community team are going to keep working on it.

For AWX I've been thinking that maybe we should just rely on manpages rather than a published guide. I know that's not quite as ideal but I think it's a good approach.

The main reason here is that it's going to take some engineering effort to create an AWX CLI guide that is separate to the Controller CLI guide. I obvs don't own the decision but I'd question if it's worth going there. IMO it's worth investing more in developing higher quality manpages than it is generating a separate AWX CLI guide. Docs should always be available from where the user is working.

Anyway, in this particular case, I suggest we just remove the link to `docs.ansible.com` because that has always been intended for downstream users and this reinforces the ambiguity between AWX and Controller imo.

Reply:
@samccann - That is fine in this context. Thanks for reminding me where the AWX kit lives!